File: Java/Estrutura condicional/aumento.java
// Importando biblioteca
import java.util.Scanner;

public class aumento {

	public static void main(String[] args) {
		
		// Variável
		double salario, porcentagem, aumento, novosalario;
		
		// Para facilitar a leitura
		Scanner sc = new Scanner(System.in);
		
		// Entrada de dados
		System.out.print("Digite o salário da pessoa: ");
		salario = sc.nextDouble();
		
		// Condicional
		 if (salario <= 1000) {
			 porcentagem = 0.2;
			 aumento = salario * porcentagem;
			 novosalario = salario + aumento;
			 System.out.print("O novo salário é de R$" + String.format("%.2f", novosalario) + ", o aumento foi de R$" + String.format("%.2f", aumento) + ", pois foi acrescentado " + porcentagem*100 + "%.");
		 }
		 
		 else if (salario <= 3000) {
			 porcentagem = 0.15;
			 aumento = salario * porcentagem;
			 novosalario = salario + aumento;
			 System.out.print("O novo salário é de R$" + String.format("%.2f", novosalario) + ", o aumento foi de R$" + String.format("%.2f", aumento) + ", pois foi acrescentado " + porcentagem*100 + "%.");
		 }
		 
		 else if (salario <= 8000) {
			 porcentagem = 0.1;
			 aumento = salario * porcentagem;
			 novosalario = salario + aumento;
			 System.out.print("O novo salário é de R$" + String.format("%.2f", novosalario) + ", o aumento foi de R$" + String.format("%.2f", aumento) + ", pois foi acrescentado " + porcentagem*100 + "%.");
		 }
		 
		 else {
			 porcentagem = 0.05;
			 aumento = salario * porcentagem;
			 novosalario = salario + aumento;
			 System.out.print("O novo salário é de R$" + String.format("%.2f", novosalario) + ", o aumento foi de R$" + String.format("%.2f", aumento) + ", pois foi acrescentado " + porcentagem*100 + "%.");
		 }
		
		sc.close(); // Boa prática de programação em Java
	}
}

File: Python/Estrutura condicional/aumento.py
# Entrada de dados
salario = float(input("Digite o salário da pessoa: "))

# Condicional
if salario <= 1000:
    porcentagem = 20/100
    aumento = salario * porcentagem
    novosalario = salario + aumento
    print(f"O novo salário é de R$ {novosalario:.2f}, o aumento foi de R$ {aumento:.2f}, pois foi acrescentado {int(porcentagem*100)}%.")

elif salario <= 3000:
    porcentagem = 15/100
    aumento = salario * porcentagem
    novosalario = salario + aumento
    print(f"O novo salário é de R$ {novosalario:.2f}, o aumento foi de R$ {aumento:.2f}, pois foi acrescentado {int(porcentagem*100)}%.")

elif salario <= 8000:
    porcentagem = 10/100
    aumento = salario * porcentagem
    novosalario = salario + aumento
    print(f"O novo salário é de R$ {novosalario:.2f}, o aumento foi de R$ {aumento:.2f}, pois foi acrescentado {int(porcentagem*100)}%.")

else:
    porcentagem = 5/100
    aumento = salario * porcentagem
    novosalario = salario + aumento
    print(f"O novo salário é de R$ {novosalario:.2f}, o aumento foi de R$ {aumento:.2f}, pois foi acrescentado {int(porcentagem*100)}%.")